impl/src/test/java/org/ehcache/WriteBehindDecoratorLoaderWriterProviderFactoryTest.java

```diff
@@ -81,6 +85,14 @@ public void testAddingWriteBehindConfigurationAtCacheLevel() {
     assertThat(serviceConfiguration, IsCollectionContaining.<ServiceConfiguration<?>>hasItem(instanceOf(WriteBehindConfiguration.class)));
     cacheManager.close();
   }
+  
+  @SuppressWarnings("unchecked")
+  @Test(expected=RuntimeException.class)
+  public void testWriteBehindWithoutCacheLoaderWriter() {
+    WriteBehindConfiguration writeBehindConfiguration = WriteBehindConfigurationBuilder.newWriteBehindConfiguration().build();
+    WriteBehindDecoratorLoaderWriterProviderFactory factory = new WriteBehindDecoratorLoaderWriterProviderFactory();
+    factory.create(mock(ServiceConfiguration.class), null).createWriteBehindDecoratorLoaderWriter(null, writeBehindConfiguration);
```

Comment:
Most of the setup in this test is superfluous as calling into the last line with nulls only will result in the same exception.
However, doing it so would require to catch the exception and make sure we are getting the exact one we expect.